opentelemetry-sdk/src/opentelemetry/sdk/_logs/_internal/__init__.py

```diff
@@ -206,6 +206,7 @@ def to_json(self, indent=4) -> str:
                 else None,
                 "dropped_attributes": self.dropped_attributes,
                 "timestamp": ns_to_iso_str(self.timestamp),
+                "observed_timestamp": ns_to_iso_str(self.timestamp),
```

Comment:
This should be `observed_timestamp`.
```suggestion
                "observed_timestamp": ns_to_iso_str(self.observed_timestamp),
```